src/artemis.cpp

```diff
@@ -52,6 +53,12 @@ Packages_t ProcessPackages(std::unique_ptr<ParameterInput> &pin) {
                         pin->GetInteger("parthenon/meshblock", "nx3")};
   artemis->AddParam("mb_dim", nb);
 
+  // Set up unit conversions for this problem
+  ArtemisUtils::Units units(pin.get(), artemis);
+  ArtemisUtils::Constants constants(units);
+  artemis->AddParam("units", units);
```

Comment:
Why pass `artemis` to the `Units` constructor? You're storing the whole thing in params, so why store other members separately as well?

Reply:
It's crucial to store HDF5-outputtable conversion factors for use in analysis scripts, we could store those conversion factors separately here and not pass the artemis package but this felt like it kept `artemis.cpp` less messy.